packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -107,58 +106,6 @@ Iterable<String> _apkFilesFor(AndroidBuildInfo androidBuildInfo) {
   return <String>['app$flavorString-$buildType.apk'];
 }
 
-/// Returns true if the current version of the Gradle plugin is supported.
-bool _isSupportedVersion(AndroidProject project) {
-  final FileSystem fileSystem = project.hostAppGradleRoot.fileSystem;
-  final File plugin = project.hostAppGradleRoot.childFile(
-      fileSystem.path.join('buildSrc', 'src', 'main', 'groovy', 'FlutterPlugin.groovy'));
-  if (plugin.existsSync()) {
-    return false;
-  }
-  final File appGradle = project.hostAppGradleRoot.childFile(
-      fileSystem.path.join('app', 'build.gradle'));
-  if (!appGradle.existsSync()) {
-    return false;
-  }
-  for (final String line in appGradle.readAsLinesSync()) {
-    if (line.contains(RegExp(r'apply from: .*/flutter.gradle')) ||
-        line.contains("def flutterPluginVersion = 'managed'")) {
-      return true;
-    }
-  }
-  return false;
-}
-
-/// Returns the apk file created by [buildGradleProject]
-Future<File> getGradleAppOut(AndroidProject androidProject) async {
-  if (!_isSupportedVersion(androidProject)) {
-    _exitWithUnsupportedProjectMessage();
-  }
-  return getApkDirectory(androidProject.parent).childFile('app.apk');
-}
-
-/// Runs `gradlew dependencies`, ensuring that dependencies are resolved and
-/// potentially downloaded.
-Future<void> checkGradleDependencies(Logger logger, ProcessUtils processUtils) async {
```

Comment:
This check is also run in the android maven cache artifacts, so it doesn't need to be repeated as a part of application package bundling